src/pages/LandingPage.tsx

```diff
@@ -29,65 +29,68 @@ import {usePaginatedGallery} from "../components/Card/usePaginatedGallery";
 
 import { GridContent } from './GridContent';
 import { ListContent } from './ListContent';
-import {usePagination} from "../hooks/usePagination";
+import { useLandingConfigStore, usePaginationStore, defaultAvailablePerPage } from '../hooks/useStore';
 import {Config} from "../config";
 
 export const LandingPage: FunctionComponent = () => {
-  const [searchInput, setSearchInput] = useState('');
-  const [view, setView] = useState<'grid'|'list'>('grid');
+  const searchInput = useLandingConfigStore((state) => state.searchInput)
+  const setSearchInput = useLandingConfigStore((state) => state.updateInput)
+
+  const selectedTags = useLandingConfigStore((state) => state.selectedTags)
+  const setSelectedTags = useLandingConfigStore((state) => state.updateSelectedTags)
+
+  const view = useLandingConfigStore((state) => state.view)
+  const setView = useLandingConfigStore((state) => state.updateView)
```

Comment:
Looks like we use every keys of the state in this component. So, I decided to just import the whole `useLandingConfigStore` state here instead of atomic approach, since it's not providing us any value here.